great_expectations/datasource/data_connector/data_connector.py

```diff
@@ -369,7 +369,7 @@ def _self_check_fetch_batch(
             )
         )
 
-        if pretty_print and df is not None:
+        if pretty_print and data is not None:
```

Comment:
Thank you so much, @cclauss -- great catch!

Reply:
@cclauss If you can, please also get the `changelog-bot` to pass its checks in Azure, and then should be good for a merge.  Thank you so much!